app/assets/javascripts/ga-events.js

```diff
@@ -41,4 +41,16 @@ $(function() {
 				});
 		});
 	});
+
+	$('[name="landing"]').each(function(){
+		$(this).click(function(e){
+				window.close()
+
+		});
+	});
+
```

Comment:
I recommend puttin this in a separate js file, or at least leave a comment so it's clear it's not related to the ga-events stuff?